src/components/map/Map.jsx

```diff
@@ -1,42 +1,40 @@
-import React from 'react';
-import { compose, withProps, withStateHandlers, lifecycle } from 'recompose';
+import React, { Component } from 'react';
+import { compose, withProps, lifecycle } from 'recompose';
 import { withScriptjs, withGoogleMap, GoogleMap } from 'react-google-maps';
 import LocationMarker from './LocationMarker';
 import config from '../../config';
+import debounce from 'lodash/debounce';
 
-const Map = compose(
+const onBoundsChangedDebouncePeriod = 500;
+const MyMap = compose(
```

Comment:
"MyMap" makes this sound like a tutorial app. :P

Reply:
I tried _Map, but the linter complained.